default.nix

```diff
@@ -130,12 +130,23 @@ let
 
       dalton = callPackage ./pkgs/apps/dalton { };
 
+      dirac = callPackage ./pkgs/apps/dirac rec {
+        stdenv = super.gcc8Stdenv;
+        exatensor = self.exatensor;
+        mpi = super.mpi.override { inherit stdenv; gfortran = super.gfortran8; };
+      };
+
       dkh = callPackage ./pkgs/apps/dkh { };
 
       dftd3 = callPackage ./pkgs/apps/dft-d3 { };
 
       ergoscf = callPackage ./pkgs/apps/ergoscf { };
 
+      exatensor = callPackage ./pkgs/apps/exatensor rec {
+        stdenv = super.gcc8Stdenv;
```

Comment:
Same here: I don't think we need to override the c-compiler. Replacing gfortran is sufficient.